build_tools/bazel/iree_c_module.bzl

```diff
@@ -52,13 +69,24 @@ def iree_c_module(
         output_to_bindir = 1,
         **kwargs
     )
-    iree_runtime_cc_library(
-        name = name,
-        hdrs = [h_file_output],
-        srcs = ["//runtime/src/iree/vm:module_impl_emitc.c", h_file_output],
-        copts = [
-            "-DEMITC_IMPLEMENTATION='\"$(location %s)\"'" % h_file_output,
-        ],
-        deps = deps,
-        **kwargs
-    )
+
+    if no_runtime:
+        iree_runtime_cc_library(
+            name = name,
+            hdrs = [h_file_output],
+            copts = [
+                "-DEMITC_IMPLEMENTATION='\"$(location %s)\"'" % h_file_output,
+            ],
```

Comment:
This deliberately has no deps?

I think it would be clearer to have a single rule and make the if branch just change what srcs and deps are, since the complicated copts bit is the same

Reply:
Done. Thanks for the suggestion.